.github/workflows/test-static-analysis.yml

```diff
@@ -0,0 +1,41 @@
+name: Test - Static Analysis
+
+on:
+  push:
+    branches: ['develop', 'feat/*', 'feature/*', 'release', 'release-candidate']
+  pull_request:
+    branches: ['develop', 'feat/*', 'feature/*', 'release', 'release-candidate']
+  schedule:
+    - cron: '30 4 * * 4'
```

Comment:
Do you think we need a cron job here? Probably makes sense to do it only for new PRs (I see that it used to work this way before).